internal/installcfg/installcfg.go

```diff
@@ -248,9 +248,19 @@ func setBMPlatformInstallconfig(log logrus.FieldLogger, cluster *common.Cluster,
 		}
 		yamlHostIdx += 1
 	}
+
+	enableMetal3Provisioning, err := common.VersionGreaterOrEqual(cluster.Cluster.OpenshiftVersion, "4.7")
+	if err != nil {
+		return err
+	}
+	provNetwork := "Unmanaged"
```

Comment:
I realize this is pre-existing, but why does assisted specify Unmanaged? I think there's been a misunderstand of its meaning.  Unmanaged = there *IS* a provisioning network but it has external DHCP infrastructure.

If you are just not enabling Metal3 at all, why not just always set it to the assisted default of `Disabled`?

Reply:
@zaneb ? @asalkeld ? 
I agree with @stbenjam it should be Disabled.